python-package/lightgbm/engine.py

```diff
@@ -408,6 +415,9 @@ def cv(params, train_set, num_boost_round=100,
     callbacks : list of callables or None, optional (default=None)
         List of callback functions that are applied at each iteration.
         See Callbacks in Python API for more information.
+    eval_train_metric : bool, optional　(default=False)
```

Comment:
Replace the char (non-breaking space?) after `optional` with normal space.

Reply:
Oh, it is mistake. .Fixed to Half-width space.